README.md

```diff
@@ -182,8 +182,6 @@ This repository only contains the source code for the package.
    export packagePAT=<GitHub Personal Access Token>
     ```
 
-To utilize the Google Calendar connector in your Ballerina application, modify the `.bal` file as follows:
```

Comment:
Is this removed intentionally?

Reply:
Yeah, it was mistakenly added in this line. It has already been correctly added here.
https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/2c4531a0d70813091099f0f6d84caaf30aeda2a1/README.md?plain=1#L87